src/hooks/useGiveInfo.ts

```diff
@@ -52,10 +53,9 @@ export const useDonationInfo = () => {
   const { activeChain = { id: 1 } } = useNetwork();
   const address = account?.address ? account.address : "";
 
-  // Hook to establish dynamic contract, meaning it will connect to the network
-  // the user is currently connected to
-  const contract = useDynamicGiveContract(GIVE_ADDRESSES, true);
+  // Establish contract
   const networks = useTestableNetworks();
+  const contract = GIVE_CONTRACT.getEthersContract(networks.MAINNET);
```

Comment:
Locking it to mainnet doesn't seem right.

Reply:
useTestableNetworks().MAINNET is of type NetworkId.MAINNET | NetworkId.TESTNET_GOERLI | NetworkId.TESTNET_RINKEBY
It returns whichever of those three networks you are connected to while matching the expected type of GIVE_CONTRACT.getEthersContract, which something like activeChain would not and restricts this to being a useful call only on the chains where YieldDirector exists

Reply:
that makes sense